information/TASK.md

```diff
@@ -57,6 +52,99 @@ export function model (): tf.LayersModel {
   return model
 ```
 
+#### Preprocessing
+
+In the Task object we can optionally choose to add preprocessing functions. Preprocessing is defined [here](../discojs/src/dataset/preprocess.ts),
+and is currently only implemented for images. The predefined functions (resize, normalize, ...) can be found there, and it is also there
```

Comment:
first keep the explanation data modality independent. what to do if i want to add a new function. how do i add dataloader and preproc code, where, and which which input/output signatures? (batchsize is also a dimension or not?)